Lib/test/test_deque.py

```diff
@@ -809,41 +814,31 @@ def test_basics(self):
         d.clear()
         self.assertEqual(len(d), 0)
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
Interesting. Do you happen to remember what the error was here? We should open an issue because of the regression.

Reply:
ERROR: test_copy_pickle (__main__.TestSubclass.test_copy_pickle)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_deque.py", line 837, in test_copy_pickle
    self.assertEqual(e.x, d.x)
AttributeError: 'Deque' object has no attribute 'x'

Reply:
thanks! I'll open the issue after merging.

Reply:
@DimitrisJim I believe this should have been broken out into a separate commit.